app/code/Transoft/Blog/Controller/Adminhtml/Index/Index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Controller\Adminhtml\Index;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\View\Result\PageFactory;
+
+/**
+ * Admin index controller
+ */
+class Index extends Action implements HttpGetActionInterface
+{
+    const ADMIN_RESOURCE = 'Transoft_Blog::blog_manage_posts';
```

Comment:
Wrong ACL. Should be **Transoft_Blog::blog_manage_items**. Fix in all controllers

Reply:
fixed